### ISSUE TITLE
Multiple screen handling issue

### ISSUE THREAD

**Issue by Ahirice** [ramboxapp/community-edition#1639]
### TL;DR

When using two screens and unplugging the second, Rambox might get unaccessible.

### Steps to reproduce

1. Be on Windows 10
2. Plug a second screen
3. Move Franz to the second screen
4. Unplug the second screen

### Expected Behavior
When I got a second screen plugged into my laptop, I can move Rambox to the second screen - currently ok
When Rambox is on the second screen, and I unplug the screen, Franz get back to the first screen

### Actual Behavior
**When Rambox  is on the second screen, and I unplug the screen, Rambox should get back to the first screen: but it doesn't, and I can't access Franz again until I close it and open it up again.**

### ENVIRONMENT

> Rambox: v0.X.X

> OS: Windows 10. Version 1709 | Build: 16299.371

> Arch: x64

**Comment by Ash258** [comment]
> and I can't access Franz again

> Move Franz

Franz?

Are you sure you have right repository?
